src/core/parse.js

```diff
@@ -24,18 +24,20 @@ export const parse = (obj, paren, wrapper) => {
         if (isImport) {
             outer += key + " " + val + ";";
         } else if (typeof val === "object") {
-
-            // Regular selector
-            let next = paren + " " + key;
-            
-            // Nested
-            if (/&/g.test(key)) next = key.replace(/&/g, paren);
```

Comment:
So, I had some time today and gave it a try. With the recent code shave, if you add a new condition just for this, that would keep the size under `1024B`.

```js
if (/,/.test(paren)) next = next.replace(/,/g, ` ${key},`);
```

What do you think?

Reply:
It seems an interesting idea, but even so you will have to decompose the `paren` and the `key`, in order to successfully nest each selector.

paren: `parent-1, parent-2` key:` &: hover, &: active, span`

```css
parent-1,
parent-2{
   color:red;
  &:active,
  &:hover,
   span{
        color:black;
  }
}
```

I have not tried the code inside goober, but I think this would happen, the expression prevents nested from being applied to the last  paren(`parent-2`) selector and only nested key to the selector that it conserved with the expression, but `&: active, span` would be badly nested... I will see how to reduce the code that I have shared